src/utils/validateSampleName.js

```diff
@@ -0,0 +1,29 @@
+const errorMessages = {
+  MIN_1_CHAR: 'Min 1 alphanumeric character',
+  INVALID_CHARS: 'Invalid characters',
+  NAME_EXISTS: 'A sample with the same name exists',
+};
+
+const validateSampleName = (input, sampleNames) => {
```

Comment:
Where is the validation difference between sample and project names coming from and why that difference?

Reply:
The project name validation is enforced so that the project name will be more verbose and descriptive, like 'Project 1'

The sample name, on the other hand, can be simpler, e.g. `WT1`, `KO`, 'WT2', because they're already scoped inside the project.

Reply:
hmm I understand now the rationale, makes sense